cmake/packaging/macos.cmake

```diff
@@ -11,7 +11,8 @@ if(SUNSHINE_PACKAGE_MACOS)  # todo
     set(INSTALL_RUNTIME_DIR "${MAC_PREFIX}/MacOS")
 
     install(DIRECTORY "${SUNSHINE_SOURCE_ASSETS_DIR}/common/assets/"
-            DESTINATION "${SUNSHINE_ASSETS_DIR}")
+            DESTINATION "${SUNSHINE_ASSETS_DIR}"
+            PATTERN "web" EXCLUDE)
```

Comment:
Not necessarily because of your PR, but should this be under `cmake/packaging/common.cmake`? It seems to be the same in all 3 files.